ros/src/tl_detector/light_classification/tl_classifier.py

```diff
@@ -19,47 +19,25 @@ class TLClassifier(object):
     https://github.com/tensorflow/models/blob/master/research/object_detection/inference/detection_inference.py
 
     """
-    def __init__(self, path_to_tensorflow_graph, confidence_thresh):
-
-        # Threshold for detections
-        self.detection_threshold = confidence_thresh
+    def __init__(self, path_to_tensorflow_graph, thresholds):
 
         # Create the TensorFlow session in which the graph is loaded
         self.session = tf.Session()
+        self.thresholds = thresholds
 
-        # Create Tensors for results
-        self.num_detections_tensor = None
-        self.detected_boxes_tensor = None
-        self.detected_scores_tensor = None
-        self.detected_labels_tensor = None
-        self.image_tensor = None
-
-        # Load the trained and frozen model graph with respective weights
-        with self.session.graph.as_default():
-            od_graph_def = tf.GraphDef()
-            with tf.gfile.GFile(path_to_tensorflow_graph, 'rb') as fid:
-                serialized_graph = fid.read()
-                od_graph_def.ParseFromString(serialized_graph)
-                tf.import_graph_def(od_graph_def, name='')
-
-                g = tf.get_default_graph()
+        with self.session.as_default():
+            saver = tf.train.import_meta_graph(path_to_tensorflow_graph+'/model.ckpt.meta')
```

Comment:
I missed the git add for that! Thanks for seeing this.

Reply:
The sim config says:
`tensorflow_graph_path: 'light_classification/inference_graph/sim/frozen_inference_graph.pb'`
whereas the site config says:
`tensorflow_graph_path: 'light_classification/fcn_graph/checkpoint'`
Will both the path to a directory containing the files as well as full path to a .pb file work with this? Looks like the simulator config would fail?